### PR TITLE
OCPBUGS-25484: Bump library-go to deal with GET conflict

### PR DESCRIPTION
**Pull request by jhadvig**
Bumping library-go to pick up https://github.com/openshift/library-go/pull/1658 and adjusting `operatorclient` pkg accordingly

/assign @stlaz 

@TheRealJon FYI

### CODE REVIEW

**Comment by stlaz** [comment]
please also include the changes from https://github.com/openshift/library-go/pull/1657

**Comment by jhadvig** [comment]
> please also include the changes from [openshift/library-go#1657](https://github.com/openshift/library-go/pull/1657)

@stlaz bumping to master

File: pkg/console/operatorclient/operatorclient.go [hunk 1]
```diff
@@ -71,3 +71,12 @@ func (c *OperatorClient) UpdateOperatorStatus(ctx context.Context, resourceVersi
 
 	return &ret.Status.OperatorStatus, nil
 }
+
+func (c OperatorClient) GetOperatorStateWithQuorum(ctx context.Context) (*operatorv1.OperatorSpec, *operatorv1.OperatorStatus, string, error) {
+	instance, err := c.Informers.Operator().V1().Consoles().Lister().Get(api.ConfigResourceName)
```
Comment (stlaz) [inline pkg/console/operatorclient/operatorclient.go:76]: this is supposed to do a live lookup

**Review by stlaz — CHANGES_REQUESTED** [review]

**Pull request merged**